src/App.js

```diff
@@ -1,20 +1,13 @@
 import React from "react";
-
-const todoList = [
-  { title: "This is a title", id: 1 },
-  { title: "Yet another title", id: 2 },
-  { title: "The last title", id: 3 },
-];
+import TodoList from "./todoList";
```

Comment:
Requires change:
This import should be importing a [TodoList.js (1st requirement)](https://github.com/Code-the-Dream-School/react/wiki/Lesson-1.2#move-list-into-new-todo-list-component) file. The reason you ran into the VSCode error is that the file is currently named as `todoList.js` and imports are case sensitive!

Reply:
@mattcarlotta fixed! Thank you for the feedback. Also, forgot to ask, but should I be making a new branch for each lesson?

Reply:
@kmg37 Yes! This should in `lesson_1_2`.

Reply:
@kmg37 You'll also have to rename the `todoList.js` file to `TodoList.js`. As mentioned on GitHub, you may have done it, but it's not being reflected in the PR.